sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuite.scala

```diff
@@ -664,6 +665,29 @@ class CastSuite extends CastSuiteBase {
     }
   }
 
+  test("SPARK-35820: Support cast DayTimeIntervalType in different fields") {
+    val ymPositive = Literal.create(
```

Comment:
`ymPositive`, I think `ym` shold be replaced. Also fix below values.

Reply:
> `ymPositive`, I think `ym` shold be replaced. Also fix below values.

DOne